engine/storage/volume/src/org/apache/cloudstack/storage/volume/VolumeObject.java

```diff
@@ -92,7 +92,7 @@ public static VolumeObject getVolumeObject(DataStore dataStore, VolumeVO volumeV
     public String getAttachedVmName() {
         Long vmId = volumeVO.getInstanceId();
         if (vmId != null) {
-            VMInstanceVO vm = vmInstanceDao.findById(vmId);
+            VMInstanceVO vm = vmInstanceDao.findByIdIncludingRemoved(vmId);
```

Comment:
@ProjectMoon I couldn't find the "findByIdIncludingRemoved" method 
(https://github.com/apache/cloudstack/blob/master/engine/schema/src/com/cloud/vm/dao/VMInstanceDao.java).

Am I missing something?